corehq/apps/app_manager/views/download.py

```diff
@@ -73,7 +75,8 @@ def download_media_suite(request, domain, app_id):
 
     """
     if not request.app.copy_of:
-        request.app.set_media_versions(None)
+        previous_version = request.app.get_latest_app(released_only=False)
```

Comment:
It's not super obvious what `request.app.get_latest_app(released_only=False)` does (it wraps `get_app`, which is itself hella complex).  Is there a more direct getter you can use in these instances?